README.md

```diff
@@ -154,10 +154,39 @@ sdk "iphoneos9.0"
 
 clean true
 
+export_options ({
+  method: "ad-hoc",
```

Comment:
Actually this shouldn't be here in the `Gymfile` section, as this is the `Gymfile` that is generated when running `gym init`